book/security.rst

```diff
@@ -264,6 +264,18 @@ Great! Now, if you go to ``/admin``, you'll see the HTTP Basic popup:
 .. image:: /images/book/security_http_basic_popup.png
    :align: center
 
+.. caution::
+
+    The ``http_basic`` firewall is only recommended while prototyping applications
+    or when the application is exclusively accesed through secure transports, such
+    as HTTPS. The reason is that browsers include user credentials in each request
+    without applying any hashing mechanism, just a plain base64 encoding.
+
+    Instead, consider using the ``http_digest`` firewall, which is almost identical
+    to ``http_basic`` but where user credentials are encoded and hashed before
+    including them in the request. Read
+    :ref:`HTTP-Digest Authentication reference <reference-security-http-digest>`.
+
```

Comment:
@javiereguiluz What was your original motivation for creating this PR? If it's security related, I think we can keep this message shorter (I'd really like to keep it as short as possible) and just highlight that `http_basic` is only secure over HTTPS and that `http_digest` may be another option over http (though it still feels funny to me to send any auth over a non-secure connection.